index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+              <a>AudioWorkletNode</a>.
+            </p>
+            <dl title="dictionary AudioParamDescriptor" class="idl">
+              <dt>
+                DOMString name
+              </dt>
+              <dd>
+                Represents the name of a parameter. An exception will be thrown
+                when the duplicated name is found when registering the class
+                definition.
+              </dd>
+              <dt>
+                float defaultValue = 0
+              </dt>
+              <dd>
+                Represents the default value of the parameter. If this value is
+                out of the range of float data type or the range defined by
+                minValue and maxValue, an exception will be thrown.
+              </dd>
+              <dt>
+                float minValue
+              </dt>
+              <dd>
+                Represents the minimum value. An exception will be thrown, if
+                this value is out of range of float data type or it is greater
+                than maxValue.
+              </dd>
+              <dt>
+                float maxValue
+              </dt>
+              <dd>
+                Represents the maximum value. An exception will be thrown, if
+                this value is out of range of float data type or it is smaller
+                than minValue.
+              </dd>
+            </dl>
+          </section>
+
+          <section>
+            <h2 id="AudioContextInfo">
+              AudioContextInfo
+            </h2>
+            <p>
+              The AudioContextInfo dictionary is used to query the current
+              status of the associated <a>AudioContext</a> within the scope of
+              an <a>AudioWorkletProcessor</a>.
+            </p>
+            <dl title="dictionary AudioContextInfo" class="idl">
+              <dt>
+                float playbackTime
+              </dt>
+              <dd>
+                The starting time of the block of audio being processed in
+                response to this event. By definition this will be equal to the
+                value of <a>BaseAudioContext</a>'s <a>currentTime</a> attribute
+                that was most recently observable in the <a>control thread</a>.
+              </dd>
+              <dt>
+                unsigned long sampleRate
+              </dt>
+              <dd>
+                Represents the sample rate of the associated
+                <a>AudioContext</a>.
+              </dd>
+            </dl>
+          </section>
+        </section>
+        <section class="informative">
+          <h2 id="AudioWorklet-Examples">
+            AudioWorklet Examples
+          </h2>
+
+          <section>
+            <h3>
+              The BitCrusher Node
+            </h3>
+            <p>
+              Bitcrushing is a mechanism by which the audio quality of an audio
+              stream is reduced - both by quantizing the value (simulating lower
```

Comment:
I don't think there should be spaces around the hyphen.  And typographically, I guess some character entity should be used. (emdash?)